### ISSUE TITLE
[BUG] Specifying background-size in background's properties breaks grapesjs

### ISSUE THREAD

**Issue by SylvainBigonneau** [artf/grapesjs#1913]
Using latest versions of `grapesjs` (0.14.55) and `grapesjs-preset-webpage` (0.1.10), I have configured my editor as such:

```javascript
import grapesjs from 'grapesjs';
import grapesJsPresetWebpage from 'grapesjs-preset-webpage';

grapesjs.init({
    // ...
    plugins: [grapesJsPresetWebpage],
    pluginsOpts: {
      [grapesJsPresetWebpage]: {
        customStyleManager: [
          {
            id: 'deco',
            name: 'Decorations',
            open: false,
            buildProps: ['background'],
            properties: [
              {
                name: 'Background',
                property: 'background',
                properties: [{
                    name: 'Background size',
                    property: 'background-size'
                  }
                ]
              }
            ]
          }
        ]
      }
    }
  });
```

The library crashes with a "TypeError: values[0] is undefined" error, as referenced in [this issue](https://github.com/artf/grapesjs/issues/1830).

If I add all the other properties in the array, such as:

```javascript
{
  property: 'background',
  properties: [
    { name: 'Image', property: 'background-image'},
    { name: 'Repeat', property:   'background-repeat'},
    { name: 'Position', property: 'background-position'},
    { name: 'Attachment', property: 'background-attachment'},
    { name: 'Size', property: 'background-size'}
  ],
}
```
the error disappears, but I don't need all of those properties...

**Comment by artf** [comment]
Ok, I think I found where is the issue, I'll fix it for the next release

**Comment by SylvainBigonneau** [comment]
Can confirm it's fixed on v0.14.57. Thanks @artf !
